### PR TITLE
Add December newsletter

### PR DESCRIPTION
**Pull request by bethaitman**
Careers section is still too long - I'll have another go at editing it down tomorrow.

Still waiting on Swapnil's blog post, which I'd like to include if he can finish it soon.

### CODE REVIEW

File: docs/blog/newsletter-december-2018.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+What a packed year it's been - with *four* conferences, and more meetups than I'm going to try to count. If you missed our most recent conference, Write the Docs Australia, there are tons of ways to catch up:
+
+- Swapnil's `blog post roundup <TODO>`_ 
+- A lovely `playlist of all the videos <https://www.youtube.com/watch?v=N13_FP2NkSk&list=PLy70RNJ7dYrJ1wANiqa7ObwUnoJjouQjt>`_ 
+- The fabulous `photo album <https://www.flickr.com/photos/writethedocs/albums/72157697741784910>`_
+
+Looking forward to next year: the Call for Proposals for Write the Docs Portland has just gone up! It's open until **midnight PST, 17 January 2019**, so get cracking and send us all your brilliant talk ideas. For full details, here's the `announcement post <http://www.writethedocs.org/conf/portland/2019/news/welcome/>`_. 
+
+The newsletter team is off next month for Christmas (it's snowing here already and I am *super* excited); we'll be back in your inboxes in February. Happy holidays, everyone!
+
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2018.rst:19]: Do you wanna lead into Slack topics, or not necessary?

File: docs/blog/newsletter-december-2018.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+There was a big discussion this month about how to define career paths for tech writers. Even the term itself was controversial, spanning two related ideas:
+
+- How a company defines levels of a role (also called  a "career ladder").
+
+  This could be levels like Associate -> Senior -> Principal/Lead, or Technical Writer Level 1 through 4 (or higher).
+
+- Where you personally go in your career.
+
+  This isn't bounded by your organisation's ladder: your path might involve changing company, or role. You might have a five-year plan, or think one step at a time, or no plan at all.
+
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2018.rst:51]: Feels to me like there is quite a bit of repetition of ladder / collab / no set path, ... if you're looking for somewhere to squish: like 48-52 AND 56 all reads like one para?

**Review by plaindocs — APPROVED** [review]
Great stuff

File: docs/blog/newsletter-december-2018.rst [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+One person pointed out that you can think of your career as a combination of the path you want and the ladder that's there: it's a collaboration between you and your company.
+
+---------------------------
+Grammar and style questions
+---------------------------
+
+This month came with a spate of language usage questions. We love getting these, so go ahead and post them in `#general <https://writethedocs.slack.com/messages/general/>`_, but other good places to check came up: 
+
+- Microsoft Manual of Style/`Microsoft Writing Style Guide <https://docs.microsoft.com/en-us/style-guide/welcome/>`_ (the usual suspect)
```
Comment (clairealyssa) [inline docs/blog/newsletter-december-2018.rst]: Is the / after Style a typo?

File: docs/blog/newsletter-december-2018.rst (hunk 1 above)
Comment (bethaitman) [reply docs/blog/newsletter-december-2018.rst:19]: I probably should - will see if I can make it not too awkward

File: docs/blog/newsletter-december-2018.rst (hunk 2 above)
Comment (bethaitman) [reply docs/blog/newsletter-december-2018.rst:51]: Good spot!

File: docs/blog/newsletter-december-2018.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-december-2018.rst]: No, but it does look weird. I'll take it out.

**Pull request merged**